### PR TITLE
Fix issue with installing happy/alex

### PR DESCRIPTION
**Pull request by jacobstanley**

### CODE REVIEW

File: src/Mafia/Install.hs [hunk 1]
```diff
@@ -362,6 +388,34 @@ createPackageSandbox penv p@(Package (PackageRef pid _ msrc) deps _) = do
 
   return ty
 
+jenkins :: Directory -> Text
```
Comment (nhibberd) [inline src/Mafia/Install.hs]: :sos:

File: src/Mafia/Install.hs [hunk 2]
```diff
@@ -362,6 +388,34 @@ createPackageSandbox penv p@(Package (PackageRef pid _ msrc) deps _) = do
...
+        hash2 = hash1 `xor` (shiftR hash1 11)
+        hash3 = hash2 + (shiftL hash2 15)
+      in
+        hash3
+  in
+    T.pack .
+    flip showHex "" .
+    loop_finish .
+    foldl' loop 0 .
+    T.unpack
```
Comment (jacobstanley) [inline src/Mafia/Install.hs]: This function is straight from Cabal, cleaned up to match our style a bit more

Comment (markhibberd) [inline src/Mafia/Install.hs]: Maybe link to the original for prosperity.

**Comment by markhibberd** [comment]
:wind_chime:

**Comment by nhibberd** [comment]
:100:

**Pull request merged**